src/core/smallmap_type.hpp

```diff
@@ -85,6 +87,16 @@ struct SmallMap : SmallVector<SmallPair<T, U>, S> {
 		return this->Find(key) != this->End();
 	}
 
+	/**
+	 * Tests whether a key is assigned in this map.
+	 * @param key key to test
+	 * @return true iff the item is present
+	 */
+	inline bool Contains(const T &key)
+	{
+		return this->Find(key) != this->End();
```

Comment:
If this starts off as `return this->count(key) > 0;` then that's one less thing to get changed later?

Reply:
Find has an early exit, count does not

Reply:
That seems to be an implementation specific thing? Either way, `this->find(key) != this->end()` then?

Reply:
I have a queued commit that does that :3